test/integration/resultsPagePlace.js

```diff
@@ -57,6 +57,12 @@ describe('The place results page', () => {
           expect($(link).attr('href')).to.have.string('https://maps.google.com');
         });
 
+        const ChoicesOverviewLinks = $('.overview a');
+        ChoicesOverviewLinks.toArray().forEach((link) => {
+          expect($(link).attr('href')).to.have.string('https://www.nhs.uk/Services/pharmacies/Overview/DefaultView.aspx');
+        });
+        expect(ChoicesOverviewLinks.length).to.equal(constants.numberOfNearbyResultsToDisplay + 1);
```

Comment:
Good idea to use the constant to save having to change it when/if that value changes. Could the `1` also be replaced with the appropriate constant?

Reply:
Done